doc/introduction/tutorials.rst

```diff
@@ -68,7 +68,6 @@ Algorithms
     /tutorials/tutorial_post_selection
     /tutorials/tutorial_boson_sampling
     /tutorials/tutorial_gaussian_boson_sampling
-    /tutorials/tutorial_machine_learning
```

Comment:
How come this was listed, yet it does not render with the previous version of docs?